### ISSUE TITLE
Cli command cannot find module 'express'

### ISSUE THREAD

**Issue by jellehak** [jeremyben/json-server-auth#46]
Cli command can't find express. Perhaps this should be added to the docs to also install `express` globally when using the global cli command `json-server-auth`.

```sh
> npx json-server-auth db.json

node:internal/modules/cjs/loader:927
  throw err;
  ^

Error: Cannot find module 'express'
Require stack:
- /usr/local/lib/node_modules/json-server-auth/dist/guards.js
- /usr/local/lib/node_modules/json-server-auth/dist/bin.js
```

**Comment by jeremyben** [comment]
It's written right at the beginning, install both json-server (which holds express) and json-server-auth.

**Comment by jellehak** [comment]
yes that's true that json-server holds express but when installing `npm i -g json-server-auth`. `json-server-auth` can't find the `express` that was installed with `json-server`. Fix I used is to also install express globally `npm i -g express`.

**Comment by jeremyben** [comment]
Did you install `json-server` globally as well when the error appeared ?

**Comment by jellehak** [comment]
Yes I have `json-server` installed globally. Problem is that nodejs doesn't resolve sibling packages from other global packages with obvious reasons.
 Possible solution would be to 
- add express as dependency
- add a `peerDependencies` to notify the user or 
- refactor to not require express at all
- or add it to the docs to install express globally.

**Comment by jeremyben** [comment]
Thank you for the detailed infos. Will look into it, and change the docs.

**Comment by nitinthewiz** [comment]
+1 on @jellehak 's solution. Installing express globally works and without, the CLI complains that express is not found. @jeremyben absolutely worth adding this to the docs. Simple and easy fix to the issue.

**Comment by OnyskivK** [comment]
I had the same issue, installed json-server in package.json file, also installed express package
But the solution in json-server version
Try "json-server --version"
if your version contain alpha it is not going to work, try to update the version, for example "json-server": "^0.17.4" and it will work

**Comment by vvalekk** [comment]
> I had the same issue, installed json-server in package.json file, also installed express package But the solution in json-server version Try "json-server --version" if your version contain alpha it is not going to work, try to update the version, for example "json-server": "^0.17.4" and it will work

This is what made it work for me. It seems that json-server-auth does not work with the 1.0 version of json-server. So install 0.17.4 instead
